src/Tickets/Commerce/Utils/Price.php

```diff
@@ -42,8 +42,8 @@ public static function sub_total( $value, $quantity, $decimal = null, $thousand_
 		$decimal      = ! is_null( $decimal ) ? $decimal : tribe( \Tribe__Tickets__Commerce__Currency::class )->get_currency_locale( 'decimal_point' );
 		$thousand_sep = ! is_null( $thousand_sep ) ? $thousand_sep : tribe( \Tribe__Tickets__Commerce__Currency::class )->get_currency_locale( 'thousands_sep' );
```

Comment:
If we need to modify these two to make sure we dont make changes to old  Tribe Commerce code, please lets do that.

Reply:
This will go away in ET-1331. Both `total` and `sub_total` will become pure methods and not deal with formatting at all.